api/src/pcapi/routes/backoffice/templates/accounts/get/details.html

```diff
@@ -29,4 +31,9 @@
       {% include "accounts/get/details/roles.html" %}
     {% endcall %}
   {% endif %}
+  {% call build_details_tab_content("activity", active_tab == 'activity') %}
+    <turbo-frame data-turbo="false" id="user_activity_frame" src="{{ url_for('backoffice_web.public_accounts.get_public_account_activity', user_id=user.id) }}">
```

Comment:
La turbo frame est chargée avec la page, alors qu'on ne souhaite faire la requête que lorsque 
```suggestion
    <turbo-frame data-turbo="false" id="user_activity_frame" loading="lazy" src="{{ url_for('backoffice_web.public_accounts.get_public_account_activity', user_id=user.id) }}">
```